db/database_sql/Makefile

```
@@ -0,0 +1,28 @@
+DBNAME:=golang_study
+DATASOURCE:=root:password@tcp(localhost:3306)/$(DBNAME)
+
+install:
+	which goose || GO111MODULE=off go get -u github.com/pressly/goose/cmd/goose
+
+docker-compose/up:
+	docker-compose up -d
+
+setup: .env install migrate/init migrate/up
+
+mysql:
+	mysql -u root -h localhost --protocol tcp -p $(DBNAME)
+
+migrate/init:
+	mysql -u root -h localhost --protocol tcp -e "create database \`$(DBNAME)\`" -p
+
+migrate/status:
+	goose status
+
+migrate/up:
+	goose up
+
+migrate/down:
+	goose down
```

githubのgooseだと必要でした。
```suggestion
migrate/status:
	goose -dir "db/migrations" mysql "$(DATASOURCE)" status
migrate/up:
	goose -dir "db/migrations" mysql "$(DATASOURCE)" up

migrate/down:
	goose -dir "db/migrations" mysql "$(DATASOURCE)" down
```